Source/Objects/Misc Objects/InFluxDB/ORInFluxDBModel.m

```diff
@@ -501,7 +501,7 @@ - (void) sendCmd:(ORInFluxDBCmd*)aCmd
     }
     [messageQueue enqueue:aCmd];
     
-    if([self connectionOK] && connectionAlarm && [connectionAlarm acknowledged]){
+    if([self connectionStatus] == kInFluxDBConnectionOK && connectionAlarm && [connectionAlarm acknowledged]){
```

Comment:
Use the new connection enums consistently. Has been giving exceptions in my tests.